inductiva/storage/storage.py

```diff
@@ -90,3 +90,18 @@ def _print_contents_table(contents):
         override_col_space=override_col_space,
         formatters=formatters,
     )
+
+
+def rmdir(dir_name: str):
+    """Deletes a directory inside the user's storage bucket. Currently,
```

Comment:
Maybe replace the bucket with `Deletes a directory inside the user's storage in Inductiva API*`? Because I am fond of exposing the GC or backend specific names in the client since it can always change.